tests/ops/integration_tests/saas/test_hubspot_task.py

```diff
@@ -186,21 +183,23 @@ async def test_hubspot_erasure_request_task(
     assert x == {
         "hubspot_instance:contacts": 1,
         "hubspot_instance:owners": 0,
-        "hubspot_instance:subscription_preferences": 1,
+        "hubspot_instance:subscription_preferences": 2,
```

Comment:
We added another subscription to the test environment so I updated the count here